app-text/tesseract/tesseract-3.04.00-r4.ebuild

```diff
@@ -92,28 +103,41 @@ src_unpack() {
 		 -execdir sh -c 'cp -- "$0" "${S}/tessdata/${0%-*}"' '{}' ';' || die
 }
 
+src_prepare() {
+	epatch "${PATCHES[@]}"
```

Comment:
We haven't ported the Java eclasses to 6 yet. Might but easy but it might not, don't know yet.